projects/laji-ui/src/lib/checkbox/checkbox.component.html

```diff
@@ -1 +1 @@
-<label [ngClass]="{'checked': isChecked, 'disabled': disabled}"><input [disabled]="disabled" type="checkbox" (input)="onInput($event)" luClickOnEnter #checkbox><ng-content></ng-content></label>
+<label [ngClass]="{'checked': isChecked, 'disabled': disabled}"><input [disabled]="disabled" type="checkbox" role="checkbox" (input)="onInput($event)" luClickOnEnter #checkbox><ng-content></ng-content></label>
```

Comment:
Adding `role="checkbox"` to actual checkbox is not needed since the input type checkbox is already semantically correct. Role is only needed if there is an element that is behaving like a checkbox, but that is not an actual checkbox input element. Also when using `role="checkbox"` the `aria-checked` attribute should also be present

Reply:
fixd